src/main/java/com/ita/if103java/ims/entity/Event.java

```diff
@@ -13,51 +13,51 @@
     private Long accountId;
     private Long warehouseId;
     private Long authorId;
-    private EventName eventName;
+    private EventName name;
     private EventType type;
```

Comment:
If you replace type with name in the DB than no need to have it in entity, in Dto let it be, but it can be got from name